src/v2/view-builder/views/ov/ChallengeOktaVerifyResendPushView.js

```diff
@@ -18,41 +18,25 @@ const Body = BaseForm.extend(Object.assign(
     },
 
     showMessages() {
```

Comment:
should this be `showCustomFormErrorCallout` instead?

Reply:
Actually `showCustomFormErrorCallout` is for errors after any user action (on form submit) when widget doesn't re-render and hence couldn't call `showMessages` (because it get called only on form.initialize). 

This is why i think you introduced this new function that adds ability to customize default courage form error callout..